dts/arm/nordic/nrf5340_cpunet.dtsi

```diff
@@ -6,6 +6,7 @@
 
 #include <arm/armv8-m.dtsi>
 #include <nordic/nrf_common.dtsi>
+#include <zephyr/dt-bindings/adc/nrf-saadc-nrf5x_v3.h>
```

Comment:
There is no SAADC in the Network core, so this is actually unneeded.

Reply:
removed